services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -387,8 +405,21 @@ async def __load_projects(
                 _check_project_permissions(row, user_id, user_groups, "read")
             except ProjectInvalidRightsError:
                 continue
+            try:
+                import concurrent.futures
+
+                with concurrent.futures.ThreadPoolExecutor() as pool:
+                    await asyncio.get_event_loop().run_in_executor(
+                        pool, ProjectAtDB.from_orm, row
+                    )
+                # ProjectAtDB.from_orm(row)
+            except ValidationError as exc:
+                log.warning("project failed validation: %s", exc)
```

Comment:
STYLE: It would be good to get some standardization in log messages.  E.g. here we know :
1. If ValidationError means there is corrupted data in the database that do not fit the expected schema
2. ValidationError does not show the original value but only where it failed
and now ask yourself, what would be the best log message that you could imaging to debug this. 
Once you have an answer, just create a helper and parametrzie all Validation errro logs

Reply:
I improved the message but I doubt you can have the same one everywhere.